code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -614,7 +614,7 @@ var/global/borer_unlock_types_leg = typesof(/datum/unlockable/borer/leg) - /datu
 		else
 			to_chat(src, "<span class='info'>You begin disconnecting from \the [host]'s nerve endings and prodding at the surface of their skin.</span>")
 
-	var/leave_time = 200
+	var/leave_time = 50
```

Comment:
5 SECONDS

Reply:
This is the same amount of time it takes to enter a host.

Reply:
No, i mean use 5 SECONDS instead of 50

Reply:
Oh, I see. Do you really think Lummox is ever going to change the unit from deciseconds, though?

Reply:
No but we already had so many cases of people accidentally making cooldowns 10 times shorter than intended
Better if whatever they're copypasting from already has SECONDS in all caps